pkg/pillar/cmd/upgradeconverter/applydefaultconfigitem.go

```diff
@@ -54,6 +67,58 @@ func applyDefaultConfigItem(ctxPtr *ucContext) error {
 		log.Errorf("Failed to Save NewConfig: %s", err)
 		return err
 	}
+	if len(keyData) != 0 {
+		os.Remove(baseAuthorizedKeysFile)
+		log.Functionf("Deleted %s file from /config/", baseAuthorizedKeysFile)
+	}
+	log.Tracef("upgradeconverter.applyDefaultConfigItem done")
+	return nil
+}
+
+func importFromConfigPartition(ctxPtr *ucContext) error {
+	var err error
+	var globalConfigPtr *types.ConfigItemValueMap
+
+	persistConfigFile := ctxPtr.oldConfigItemValueMapFile()
+	globalConfigExists := fileExists(globalConfig)
+
+	if globalConfigExists {
+		globalConfigPtr, err = parseFile(globalConfig)
+		if err != nil {
+			log.Errorf("Error parsing configuration from file: %s, %s", globalConfig, err)
+			return err
+		}
+	} else {
+		log.Noticef("No existing ConfigItemValueMap; creating new %s",
+			persistConfigFile)
+		globalConfigPtr = types.NewConfigItemValueMap()
+	}
+	keyData := readAuthorizedKeys(baseAuthorizedKeysFile)
```

Comment:
Ah you have it here but didn't remove it from the other function.

Reply:
Sorry. Last commit was not a complete one. I was trying to sync my sandbox between linux & mac machines. Diff gave issues and I resorted to commit  and then pull from different machine. Forgot that I already have a PR open for it.